### PR TITLE
[Minor] doc: Add good first issue and updated text on working on an issue.

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Add good first issue and updated text on working on an issue.

### Why are the changes needed?

Make people aware of goof first issues and that they don't need to ask for permission to fix something.

Fix: # N/A

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Build locally.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,9 +53,13 @@ If you find a bug in Gravitino, please open an issue on GitHub. Be sure to inclu
 
 If you have ideas for enhancements or new features, feel free to create an issue to discuss them. Gravitino welcomes suggestions and provides prompt feedback on their feasibility and relevance.
 
+### Good first issues
+
+If you are new to open source or can't find something to work on check out the [Good First Issues list](https://github.com/datastrato/gravitino/contribute).
```
Comment (qqqttt123) [inline CONTRIBUTING.md:58]: Should we use `https://github.com/datastrato/gravitino/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22`?

Comment (justinmclean) [reply CONTRIBUTING.md:58]: No https://github.com/datastrato/gravitino/contribute is a better link and lists all good first issues.

Comment (justinmclean) [reply CONTRIBUTING.md:58]: It also links to the contribution guide.

Comment (qqqttt123) [reply CONTRIBUTING.md:58]: OK, I just clicked the link. It went wrong. But I clicked again. It's ok now.

**Review by jerryshao — APPROVED** [review]
LGTM, thanks @justinmclean for your contribution.

**Pull request merged**